webboot/webboot.go

```diff
@@ -8,9 +8,42 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"log"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+	"github.com/u-root/webboot/pkg/webboot"
+)
+
+var (
+	cmd       = flag.String("cmd", "", "Command Line")
+	mountDir  = flag.String("dir", "/tmp/mountDir", "The mount point of the ISO")
+	osSystems = map[string]webboot.Distro{
+		"TinyCore": webboot.Distro{"/boot/vmlinuz64 ", "/boot/initrd", "--reuse-cmdline", "https:louis.com"},
+		"Ubuntu":   webboot.Distro{"/boot/vmlinuz ", "/boot/init", "--reuse-cmdline", "https:louis.com"},
+	}
 )
 
 func main() {
-	fmt.Println("weboot up")
+
+	flag.Parse()
+	if len(flag.Args()) < 1 {
+		log.Fatal("error:pass in an operating system")
+	}
+
+	osname := flag.Args()[0]
+
+	if _, ok := osSystems[osname]; !ok {
+		log.Printf("error operating system is not supported")
```

Comment:
I'm wondering why you have two log.Printf instead of one?

Reply:
I wanted to make sure the error output is clear and easily readable by the user.